static/update.sh

```diff
@@ -21,6 +21,12 @@ mkdir -p "$SCRIPTS"
 if [ "${1}" = "minor" ]
 then
     echo "$((NCMAJOR-1))" > /tmp/minor.version
+elif [ "${1}" = "beta" ]
+then
+    echo "beta" > /tmp/prerelease.version
+elif [ "${1}" != "" ]
```

Comment:
Add a value like "latestbeta" or something, looking for nothing is dangerous here.

Reply:
the thing was, that I wanted to enable the people to enter a version like 19.0.0RC1 here...
I think all other cases are catched by the latest beta version...

Reply:
I am not looking for nothing but for everything then nothing...

Reply:
or do you have a better idea how to allow something like 16.0.0RC1?

Reply:
`grep` for RC at least :)

Reply:
is fixed 👍